R/convenience.R

```diff
@@ -20,11 +20,11 @@ NULL
 #' @rdname ReadAkoya
 #'
 LoadAkoya <- function(
-  filename,
```

Comment:
I recommend removing all of the space-only changes so that reviewers can focus on changes that actually make a functional difference. Thank you so much for all of your work on this!

Reply:
ok, I think most of the space-only changes are now removed. Thanks